app/main.ipynb

```diff
@@ -31,15 +35,29 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
+   "execution_count": 478,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "False\n"
+     ]
+    }
+   ],
    "source": [
-    "# write your code for task 0 here"
+    "dataset = pd.read_csv(\"../data/adult.csv\", na_values=\"?\")\n",
+    "correct_format_of_salary = dataset[\"salary\"].str.match(r\"\\d+[Kk]\").all()\n",
+    "print(correct_format_of_salary)"
```

Comment:
you didn't implemented this "Also check for data correctness (salary & salary $K)."